README.md

```diff
@@ -65,83 +82,107 @@ A linter is a tool that checks your code for:
+file with the local `render` method imported. The import should look something
+like this.
+
+```js
+import { render } from '../../test-utils';
+```
 
 ## 📖 Useful resources
 
-- The [docs](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#table-of-contents) for `create-react-app` contain a lot of solutions for common problems when writing a React application.
+- The [docs](https://create-react-app.dev/docs/documentation-intro) for
+  `create-react-app` contain a lot of solutions for common problems when writing
+  a React application.
 - Foundry [docs](https://github.com/sumup/foundry#table-of-contents).
-- Circuit UI [docs](https://circuit.sumup.com/#/) and [storybook](https://circuit.sumup.com/storybook/).
-- The [README](https://github.com/kentcdodds/react-testing-library/blob/master/README.md#what-is-react-testing-library) for`react-testing-library`.
+- Circuit UI [docs](https://circuit.sumup.com/#/) and
+  [storybook](https://circuit.sumup.com/storybook/).
+- The [README](https://testing-library.com/docs/react-testing-library/intro/)
+  for`react-testing-library`.
 
 ## 💁‍♀ Tips
 
 ### Setting up Foundry's ESLint in your editor
 
-The great strength of using a toolkit like Foundry is that you as a user do not have to care about developer dependencies like ESLint. Foundry manages and runs them for you via its `run` command. This, however, becomes a bit tricky when you want your text editor or IDE to run ESLint on your code as you edit it. To get this working, you need to tell your editor where to look for ESLint dependencies.
+The great strength of using a toolkit like Foundry is that you as a user do not
+have to care about developer dependencies like ESLint. Foundry manages and runs
+them for you via its `run` command. This, however, becomes a bit tricky when you
+want your text editor or IDE to run ESLint on your code as you edit it. To get
+this working, you need to tell your editor where to look for ESLint
+dependencies.
 
-Your mileage may vary depending on your text editor, but we try to keep a list of instructions here.
+Your mileage may vary depending on your text editor, but we try to keep a list
+of instructions here.
 
 #### Visual Studio Code
 
-The ESLint extension for VSCode has a `nodePath` setting. It allows you to add additional paths where the plugin should look when resolving dependencies. By setting it to the default path of Foundry in a project's `node_modules`, you will make VSCode work with Foundry's version of ESLint. Put the following line in your `settings.json`.
+The ESLint extension for VSCode has a `nodePath` setting. It allows you to add
+additional paths where the plugin should look when resolving dependencies. By
+setting it to the default path of Foundry in a project's `node_modules`, you
+will make VSCode work with Foundry's version of ESLint. Put the following line
+in your `settings.json`.
 
```

Comment:
I don't have this set in my VS Code and it works just fine. Is this still necessary?

Reply:
Not sure. I was mostly not sure which troubleshooting items to keep and which not. I would have liked to just delete all of them. 😅

Regarding the actual problem: I think there may be cases where module resolution fails to pick up foundry's ESLint because multiple versions of ESLint being installed. I had this problem with the pre-4.0 version of Foundry. However, in that case even the CRA build fails, so probably there is no need to do this. I'll get rid of the section.